controllers/users.controller.js

```diff
@@ -0,0 +1,14 @@
+const { retrieveUsers } = require("../models/users.models.js")
+
+function getUsers(req, res, next) {
+    const { query } = req
+    return retrieveUsers(query)
+    .then(({rows}) => {
+        res.status(200).send(rows)
```

Comment:
We want to be sending back our response on an object on a descriptive key like 'users' for example